frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.tsx

```diff
@@ -0,0 +1,126 @@
+import { FastSelect, SelectOptions } from 'components/common/form';
+import { Label } from 'components/common/Label';
+import TooltipIcon from 'components/common/TooltipIcon';
+import { CLASSIFICATIONS } from 'constants/classifications';
+import { useFormikContext } from 'formik';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+/**
+ * component responsible for displaying classifcations of properties with conditional renders of information based on the classification selected.
+ */
+
+const Title = styled.h4`
+  float: left;
+`;
+
+/** formated information box to display the classification definitions to the right of the select */
+const InfoBox = styled.div`
+  border: 1px solid #f2f2f2;
+  border-radius: 4px;
+  text-align: left;
+  padding: 8px 12px 10px;
+`;
+
+/** formatted field description that will appear underneath the select input */
+const FieldDescription = styled.p`
+  margin-left: 152px;
+  font-size: 12px;
+  width: 250px;
+  text-align: left;
+`;
+
+/** used to conditionally render the information display box with the desired content */
+const InfoBoxWithContent = (content: string) => {
+  return (
+    <InfoBox>
+      <p style={{ fontSize: '11px' }}>{content}</p>
+    </InfoBox>
+  );
+};
+
+interface IClassificationFormProps {
+  /** title to be displayed at the top of the form */
+  title?: string;
+  /** the classification set to be used with the component, could vary by form */
+  classifications: SelectOptions;
+  /** the field label for the select options (eg) Building Classification/Land Classification */
+  fieldLabel?: string;
+  /** pass down the parents nameSpace in order to access the appropriate fields */
+  nameSpace?: string;
+  /** pass the index to access the appropriate property if it is in a list format */
+  index?: string;
+  /** pass a tooltip for the classification field */
+  toolTip?: string;
+}
+
+export const ClassificationForm: React.FC<IClassificationFormProps> = ({
+  title,
+  classifications,
```

Comment:
Classifications currently unfiltered. Will be updated later with form implementation

Reply:
I think we are allowing users to select any classification other than `Disposed`